src/index.hot.js

```diff
@@ -1,19 +1,20 @@
+/* eslint global-require: 0 */
+import 'babel-polyfill';
```

Comment:
hmmm, may i ask how do you set up for your dev webpack config.js

I try to keep prod and dev as similar, but i still get the error .
```
rootSaga.js?e099:3 Uncaught ReferenceError: regeneratorRuntime is not defined
    at eval (eval at ./Root/rootSaga.js (main.js:5596), <anonymous>:8:30)
    at Object../Root/rootSaga.js (main.js:5596)
    at __webpack_require__ (main.js:658)
    at fn (main.js:84)
```

if I don't import `babel-polyfill`.